microsoftgraph/microsoft-graph-explorer-v4
# Fix: Add mgt link colors

## Overview

fixes #2227 

### Demo

![image](https://user-images.githubusercontent.com/18407044/202442489-9684d14b-4f01-468c-9ad5-ad4e4fdb3ac5.png)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Checkout branch
* Load Graph Explorer
* Run 'my profile (beta)'
* Navigate to the Toolkit component tab
* Observe link fix
